src/app/shell/icons/Library.ts

```diff
@@ -110,7 +110,7 @@ export {
   faCheckCircle as redeemedIcon,
   faCheckCircleRegular as completedIcon,
   faCheckCircleRegular as unselectedCheckIcon,
-  faChevronCircleDown as openDropdownIcon,
+  faEllipsisV as openDropdownIcon,
```

Comment:
We use this icon in a couple of other places - probably best to make a separate export.

Reply:
oof, good catch. thanks!